### PR TITLE
Archs/alvaro/create wonders

### PR DESCRIPTION
**Pull request by junior0123**
Hi Jose, I fixed the names that are not understood, I finished completing all the wonders and put them in a folder. I had a slight problem when making the commits, because when I added to the stage I added files that I had to make in different commits, and they were made in one.

### CODE REVIEW

File: src/main/java/jalau/at18/architects/Wonder.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package jalau.at18.architects;
+
+public abstract class Wonder {
+    private final int maxOfStages = 5;
+    private Stage[] architects;
```
Comment (J0c4) [inline src/main/java/jalau/at18/architects/Wonder.java:5]: Why the list of stages is named `architects?`

File: src/main/java/jalau/at18/architects/StagesEphesus.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+enum StagesEphesus {
+    ONE(2, false, 3),
+    TWO(2, true, 3),
+    THREE(3, false, 4),
+    FOUR(3, true, 5),
+    FIVE(4, false, 7);
+
+    private int countOfResources;
+    private boolean equals;
```
Comment (J0c4) [inline src/main/java/jalau/at18/architects/StagesEphesus.java:11]: `countOfResources` and `points` are easy to understand, however, I'm not fully clear on `equals`, what does this attribute means?

**Review by J0c4 — CHANGES_REQUESTED** [review]
Looks good, please check my comments, mainly questions. 

Please do rebase so we can approve/merge

File: src/main/java/jalau/at18/architects/Wonders/Stage.java [hunk 3]
```diff
@@ -1,21 +1,21 @@
-package jalau.at18.architects;
+package jalau.at18.architects.Wonders;
```
Comment (J0c4) [inline src/main/java/jalau/at18/architects/Wonders/Stage.java:1]: Please use all lower case for package names, this helps to not confuse them with classes whose names start with a capital letter. `wonders`

Folder name will need to be renamed as well

**Review by J0c4 — CHANGES_REQUESTED** [review]
Thanks for the updates @junior0123 , those look good. Just one observation about the package name, please fix that and do rebase, then we will be able to merge this

File: src/main/java/jalau/at18/architects/Wonders/StagesHal.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package jalau.at18.architects.Wonders;
+enum StagesHal {
```
Comment (FernandezRobert) [inline src/main/java/jalau/at18/architects/Wonders/StagesHal.java:2]: You probably meant "StagesHall"?

Comment (junior0123) [reply src/main/java/jalau/at18/architects/Wonders/StagesHal.java:2]: no, it is an abbreviation of "Halicarnassus", as the name was too long.

**Review by J0c4 — CHANGES_REQUESTED** [review]
Pending to rename wonders folder with lowercase

**Comment by junior0123** [comment]
> Pending to rename wonders folder with lowercase
the folder problem was fixed using the command 
- git mv <Folder name> tmp
- git mv tmp <folder name>
and the rebase was done

**Review by J0c4 — APPROVED** [review]

**Pull request merged**